src/audiomixerboard.cpp

```diff
@@ -995,6 +998,9 @@ void CAudioMixerBoard::HideAll()
     // use original order of channel (by server ID)
     ChangeFaderOrder ( ST_NO_SORT );
 
+    // Reset recording indication styleSheet
```

Comment:
I was thinking of a class variable to hold the current stylesheet based on the GUI design, set in a switch here.

Then, in UpdateTitle, set the stylesheet from the class variable if recording is active, else set the stylesheet to `""`.

Reply:
There is no skin dependency when setting here in CAudioMixerBoard. When the styleSheet is reset with "" it reverts to the styleSheet of the underlying class which has already been set according the the skin.

Reply:
OK - it was just the previous version seemed to have different strings per GUI design.  If it's not required, this is fine.

The only other thing might be to put the value at 1109 into a `const` or `#define` in the header, rather than in line.